### DISCUSSION TITLE
Grundsätzliche Einstellungen

### DISCUSSION THREAD

**Discussion by rieders** [hoylabs/OpenDTU-OnBattery#339] [General]
Hallo

Ich versuche seit wenigen Tagen mit openDTU Battery zurerecht zu kommen.
Heute habe ich die DTU fertiggestellt.
Mein SDM 630 Stromzähler ist per MQTT über Tasmota verbunden.
Nun habe ich versucht diesen in die DTU einzubinden. MQTT Explorer gibt mir diesen Pfad 

Topic Smartmeter_Garten/stat/STATUS10
 oder Smartmeter_Garten/tele/SENSOR

{"StatusSNS":{"Time":"2023-07-25T17:42:49","SDM630":{"Voltage_P1":231.08,"Voltage_P2":234.49,"Voltage_P3":231.89,"Current_P1":0.08,"Current_P2":0.11,"Current_P3":0.00,"Power_P1":0.00,"Power_P2":-8.50,"Power_P3":0.00,"Current":0.13,"Total":622.490,"Power":-6.91,"Import_nres":240.486,"Export_nres":382.004,"Export_P1":23.571,"Export_P2":358.427,"Export_P3":0.006,"Import_P1":20.664,"Import_P2":182.474,"Import_P3":37.348,"L1_total_kwh":44.235,"L2_total_kwh":540.901,"L3_total_kwh":37.354,"Blindleistung_L1":-18.418,"Blindleistung_L2":-24.649,"Blindleistung_L3":0.000,"Scheinleistung_L1":18.546,"Scheinleistung_L2":26.098,"Scheinleistung_L3":0.000,"VAh_since_last_reset":708.504,"1":53,"1":286}}}

Wobei ich sagen muss das ich nicht den Standart-Port 1883 sondern 1885 benutze.

Eine andere Sache. Ich benutze Victron Ladegerät als auch Generic Venus Device als Raspberry Pi Version und ein Smart Shunt 500A/50.
Kann ich mit dem v.bus von der DTU zum Venus?

Ich würde mich freuen wenn mir jemand weiter helfen kann

Grüße

**Comment by spcqike** [comment]
Für das powermeter empfehle ich dir die Konfiguration als http(s)+json. Kein MQTT. 

Lass die opendtu dein Tasmota direkt abfragen. So habe ich es auch 

Als URL

    http://ip/cm?cmnd=status%208

als json pfad 

    StatusSNS/SDM630/Power

zum ve.Bus kann ich nichts beitragen 😄

**Reply by johi-1** [reply]
Hallo, hatte die gleiche Frage und habe deine Lösung versucht. Funktioniert.
danke
